### PR TITLE
chore(webapp): update web-ssh-gui (new Rust stack)

### PR DESCRIPTION
**Pull request by irvingoujAtDevolution**

### CODE REVIEW

File: webapp/src/client/app/modules/web-client/ssh/web-client-ssh.component.ts [hunk 1]
```diff
@@ -180,15 +180,18 @@ export class WebClientSshComponent extends WebClientBaseComponent implements OnI
...
+      this.remoteTerminal.connect({
+        hostname: connectionParameters.host,
+        port: connectionParameters.port,
+        username: connectionParameters.username,
+        proxyUrl: connectionParameters.gatewayAddress + `?token=${connectionParameters.token}`,
+        passpharse: connectionParameters.privateKeyPassphrase ?? '',
+        privateKey: connectionParameters.privateKey ?? '',
+        password: connectionParameters.password ?? '',
+        onHostKeyReceived: (serverName, fingerprint) => {
+          return Promise.resolve(true);
```
Comment (irvingoujAtDevolution) [inline webapp/src/client/app/modules/web-client/ssh/web-client-ssh.component.ts:192]: To be completed in separate PR

**Review by CBenoit — APPROVED** [review]
LGTM!
